EMIP/1-1000/EMIP00816.xml

```diff
@@ -29,34 +29,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <collection>EMIP</collection>
                   <idno facs="EMIP/Codices/816/">Chelekot Silasie 75</idno>
                </msIdentifier>
+
                <msContents>
-                  <summary>Homilies on Right Faith, ድርሳነ፡ ርቱዓ፡ ሃይማኖት።; Homily on Wälätä Maryam by Hǝrǝyaqos, ድርሳነ፡ ወለተ፡ ማርያም፡</summary>
-                  <msItem xml:id="ms_i1">
-                     <textLang mainLang="gez">Gǝ‘ǝz</textLang>
-                  </msItem>
+                  <summary>Homilies on Right Faith, ድርሳነ፡ ርቱዓ፡ ሃይማኖት።; Homily on Walatta Māryām by Hǝrǝyāqos, ድርሳነ፡ ወለተ፡ ማርያም፡</summary>
+
+<msItem xml:id="ms_i1">
+<locus from="1r" to="136r" facs="001"/>
```

Comment:
If you are working from the PDF that I have, then 1r does not have image number 001. As there is an initial page with some scrawls on, 1r corresponds to facs="002".